Messaging/Forward Messages to Email - Node.js/index.js

```diff
@@ -0,0 +1,48 @@
+require("dotenv").config();
+let sw = require("@signalwire/realtime-api");
+let Messaging = sw.Messaging;
+let formData = require("form-data");
+let mailgun = require("mailgun.js");
+
+const swClient = new Messaging.Client({
+    project: process.env.PROJECT_ID,
+    token: process.env.API_TOKEN,
+    contexts: ["office"]
+});
+
+const Mailgun = new mailgun(formData);
+const mgClient = Mailgun.client({ username: 'api', key: process.env.MAILGUN_API_TOKEN });
```

Comment:
the username "api" is hardcoded here; is this correct? Or should it go in the env file?

Reply:
https://documentation.mailgun.com/en/latest/quickstart-sending.html#send-via-api
The Mailgun documentation recommends this setup